<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>快递管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rec.qrc">
    <normaloff>:/new/prefix1/icon.ico</normaloff>:/new/prefix1/icon.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>查询</string>
       </attribute>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>260</y>
          <width>156</width>
          <height>23</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>快递号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="orderkey">
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>公司</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="company">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>结果</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>401</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="sender"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>联系方式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="senderphone"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>收件人</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="receiver"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>联系方式</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="recieverphone"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>定单号</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="orderkey1"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>订单日期</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="orderdate"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>邮递员</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="postman"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>联系方式</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="postphone"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>到达站</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="stop"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>位置</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="address"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="senders">
           <property name="text">
            <string>发件人</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>77</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>继续查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>高级</string>
       </attribute>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>查询结果</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>213</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>按姓名查找</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="dependname"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="dependstop"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="dependate"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>按站查找</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>按日期查找</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>256</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>新建</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>441</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>发件人地址</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="saddress">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>收件人地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="raddress">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>250</y>
          <width>158</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>提交</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>45</y>
          <width>441</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="4" colspan="2">
          <widget class="QLineEdit" name="rphone">
           <property name="maxLength">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="recieverman">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QLineEdit" name="sphone">
           <property name="maxLength">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>收件人</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>手机号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>手机号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>发件人</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="senderman">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>承运公司</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4" colspan="2">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>开始站</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>职工</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>231</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>快递员</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>195</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="pname">
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>电话</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pphone"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>所属公司</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="pcompany"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>性别</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="psex"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>221</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>职员</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>177</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ename">
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>电话</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ephone"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>服务站</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="estop"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>职位</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="eposition"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputp">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>260</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入姓名以查询</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="inpute">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>260</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入姓名以查询</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>高级</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_2"/>
   <addaction name="action_7"/>
   <addaction name="disconnect"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/close.ico</normaloff>:/new/prefix1/close.ico</iconset>
   </property>
   <property name="text">
    <string>连接数据库</string>
   </property>
  </action>
  <action name="disconnect">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/open.ico</normaloff>:/new/prefix1/open.ico</iconset>
   </property>
   <property name="text">
    <string>断开数据库</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/about.ico</normaloff>:/new/prefix1/about.ico</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/search.ico</normaloff>:/new/prefix1/search.ico</iconset>
   </property>
   <property name="text">
    <string>高级查询</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/employee.ico</normaloff>:/new/prefix1/employee.ico</iconset>
   </property>
   <property name="text">
    <string>职工信息</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/order.ico</normaloff>:/new/prefix1/order.ico</iconset>
   </property>
   <property name="text">
    <string>新建订单</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
